Library/Formula/sfnt2woff.rb

```diff
@@ -0,0 +1,18 @@
+class Sfnt2woff < Formula
+  desc "Convert TTF and OTF fonts to WOFF format."
+  homepage "https://people.mozilla.org/~jkew/woff/"
+  url "https://people.mozilla.org/~jkew/woff/woff-code-latest.zip"
```

Comment:
The only path of action that would meet our guidelines and user expectation is for upstream to tag a versioned release. A lot of upstream devs understand the packaging issue and are willing to tag a versioned release when asked, so that is probably worthwhile doing.